nextflow.config

```diff
@@ -19,6 +21,7 @@ params {
 
     // Genome
     genome                     = 'GRCh38'
+    genomes                    = null
```

Comment:
Is this variable used somewhere? I can't find it.

Reply:
It is not indeed! But when not defined there is a warning